src/main/java/com/tournament/cup/settings/DatabaseClass.java

```diff
@@ -0,0 +1,80 @@
+package com.tournament.cup.settings;
+
+import com.tournament.cup.play.Squad;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+import java.sql.*;
+
+@Repository
+@AllArgsConstructor
+public class DatabaseClass {
+    private static final String URL = "jdbc:mysql://localhost:3306/cup";
+    private static final String USER = "root";
+    private static final String PASSWORD = "root";
+
+
+    public void teamAuthorization(String teamName, String coach, String captain) {
+        Integer points = 0;
+        try (Connection connection = DriverManager.getConnection(URL, USER, PASSWORD)) {
+            PreparedStatement state = connection.prepareStatement("INSERT INTO teams(team_name, captain, coach,points) VALUES (?,?,?,?)");
+            state.setNString(1, teamName);
+            state.setNString(2, captain);
+            state.setNString(3, coach);
+            state.setInt(4, points);
+            state.executeUpdate();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void showTable() {
+        Integer number = 0;
+        String query = "SELECT team_name,points FROM teams ORDER BY points DESC ;";
+
+        try {
```

Comment:
Avoid try block in another try block.
This piece of code can be done in single try block.